web_app.py

```diff
@@ -38,6 +38,7 @@
     "/api/choropleth/geojson?placeDcid=country/USA&placeType=County",
     "/api/place/parent/country/USA",
     "/api/place/places-in-names?dcid=country/USA&placeType=County",
+    "/api/choropleth/geojson?placeDcid=Earth&placeType=Country",
```

Comment:
nit, move this above (so sort all endpoints here)

Reply:
done